web/template/certificate_provider_check_your_name.gohtml

```diff
@@ -0,0 +1,54 @@
+{{ template "page" . }}
+
+{{ define "pageTitle" }}{{ tr .App "checkAndConfirmYourName" }}{{ end }}
+
+{{ define "main" }}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-xl govuk-!-margin-bottom-6">{{ tr .App "checkAndConfirmYourName" }}</h1>
+
+            {{ trFormatHtml .App "checkAndConfirmYourNameContent" "DonorFullName" .Lpa.You.FullName "CertificateProviderFullName" .Lpa.CertificateProvider.FullName }}
+
+            <form novalidate method="post">
+                <div class="govuk-form-group {{ if .Errors.Has "is-name-correct" }}govuk-form-group--error{{ end }} govuk-!-margin-bottom-6">
+                    <fieldset class="govuk-fieldset">
+                        {{ template "error-message" (errorMessage . "is-name-correct") }}
```

Comment:
I was going to say this should go below the legend, but looks like the CSS makes it happen anyway TIL

Reply:
Yeah I was super confused for a second there until I saw what was going on.